File: C/fizzBuzz.c
/*
 * fizzBuzz.c by Jigsy (https://github.com/Jigsy1) released under the Unlicense.
 *
 * Compiled on Windows 8.1 using TCC available from: https://bellard.org/tcc/
 *
 * How to:
 * ----------
 * ...> tcc -c fizzBuzz.c
 * ...> tcc -run fizzBuzz.o
 *
 */

#include <stdio.h>

#define buzz "Buzz"
#define fizz "Fizz"
#define fizzBuzz "Fizz Buzz"

int main() {
	int thisLoop;
	for (thisLoop = 1; thisLoop <= 100; thisLoop++) {
		if (thisLoop % 5 == 0 && thisLoop % 3 == 0) {
			printf("%s, ", fizzBuzz);
			continue;
		}
		else if (thisLoop % 5 == 0) {
			printf("%s, ", buzz);
			continue;
		}
		else if (thisLoop % 3 == 0) {
			printf("%s, ", fizz);
			continue;
		}
		else {
			printf("%d, ", thisLoop);
		}
		// The output symmetry is quite interesting.
	}
	return 1;
}

// EOF

File: py3/fizzBuzz.py
# fizzBuzz.py by Jigsy (https://github.com/Jigsy1) released under the Unlicense.

for thisLoop in range(1, 101):
	# `-> Start at 1, end at 100.
	if thisLoop % 5 == 0 and thisLoop % 3 == 0:
		print("Fizz Buzz")
		continue
	if thisLoop % 5 == 0:
		print("Buzz")
		continue
	if thisLoop % 3 == 0:
		print("Fizz")
		continue
	print(thisLoop)

# EOF